src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes`. In this case, you've styled `.stars__star` correctly with a class, but make sure all other elements follow this guideline as well.